.github/workflows/test.yml

```diff
@@ -78,6 +83,12 @@ jobs:
           override: true
           profile: minimal
           default: true
+      - name: Downgrade dependencies for older Rust versions
+        shell: msys2 {0}

```

Comment:
Note that this way of invoking the commands will not fail the build on error. In one test, the `cargo update` failed to run, but the build kept going and failed because the dependencies were too new.